docs/internal/release/release-process.md

```diff
@@ -0,0 +1,38 @@
+# Release process.
+
+## Regular release.
+
+This is a regular flow, when you are just following our usual versioning and there no additional conditions.   
+
+For a `next` release, run Release workflow from Github actions panel on `development` branch.
+For a stable release, do the following:
+
+1. Create branch from `development` for next minor/major as `release/v1.2.x`.
+2. Update `libs/lerna.json` version to a new minor/major version as `1.2.0`.
+3. Push the release branch.
+4. Trigger Release workflow from Github actions panel on release branch.
```

Comment:
Should it be done manually? 

We can have a rule like in https://github.com/spryker/oryx/blob/bb526e9cf991a7b751b920ec26ac484e690a45d5/.github/workflows/test.yml#L7 to make it automatically on branches starting with `release/` to make it automatic

Reply:
Yep, probably we can do it. Don't see any issues for now.